test/README.md

```diff
@@ -58,10 +58,19 @@ environment.
    See [Installing Helm](https://helm.sh/docs/intro/install/) for other options.
    Tested with version v3.11.
 
-1. Install the `virtctl` tool. See
+1. Install the `virtctl` tool.
+   Working Example:
+   ```
+   curl -L -O https://github.com/kubevirt/kubevirt/releases/download/v1.2.0/virtctl-v1.2.0-linux-amd64
+   ```
+   After download completes for `virtctl` issue this command.
+
+   ```
+   sudo install virtctl /usr/local/bin
```

Comment:
I tested this and it does not work - because the curl command downloads `virtctl-v1.2.0-linux-amd64` and not `virtctl`.

After installing, we are left with unwanted executable in the download location.

See the mc instructions bellow for similar instructions without these issues.